pages/level/[username]/[slugName].tsx

```diff
@@ -124,6 +127,56 @@ function LevelPage() {
     router.push(nextUrl);
   };
 
+  const [records, setRecords] = useState<Record[]>();
+
+  const getRecords = useCallback(() => {
+    if (!level) {
+      return;
+    }
+
+    fetch(`/api/records/${level._id}`, {
+      method: 'GET',
+    }).then(async res => {
+      if (res.status === 200) {
+        setRecords(await res.json());
+      } else {
+        throw res.text();
+      }
+    }).catch(err => {
+      console.error(err);
+      alert('Error fetching records');
```

Comment:
toast.dismiss(); toast.errror(… maybe instead of alert like we do for the other places?